docs/source/guides/faq.md

```diff
@@ -0,0 +1,121 @@
+
+#### How can I host my schema online?
+
+A great tool for learning and building small projects is [Glitch](https://glitch.com). Glitch allows development of a schema in the browser, and even supports cloning from and pushing to GitHub. Glitch provides a public endpoint that projects can query against. To get started with building a GraphQL schema, try using and remixing the [Apollo Launchpad](https://glitch.com/~apollo-launchpad) project.
+
+GraphQL schemas written with `apollo-server` can be deployed anywhere that other Node.js projects can be deployed. `apollo-server` even has variants to support serverless deployment with AWS Lambda.
+
+There are deployment guides currently written for [Heroku](https://www.apollographql.com/docs/apollo-server/deployment/heroku.html), [Lambda](https://www.apollographql.com/docs/apollo-server/deployment/lambda.html), and [Now](https://www.apollographql.com/docs/apollo-server/deployment/now.html)
+
+#### How do I connect my client app to my schema?
+
+There are many tools available to connect any kind of client to a schema. The most common are [Apollo Client](https://www.apollographql.com/docs/react/) for JavaScript clients,
+[Apollo iOS](https://www.apollographql.com/docs/ios/) for native iOS clients, and [Apollo Android](https://github.com/apollographql/apollo-android) for native Android clients.
+
+For Apollo Client projects, there are also many view-layer integrations, to make querying GraphQL schemas easier in [React](https://www.apollographql.com/docs/react/essentials/get-started.html), [Vue](https://github.com/Akryum/vue-apollo), and [Angular](https://www.apollographql.com/docs/angular/).
+
+## Building a proof of concept
+
+> People in this phase understand how GraphQL works, and what benefits it offers. They are trying to create a proof of concept for their projects or their company to test GraphQL's viability in production
+
+#### How do I wrap existing APIs?
+
+One of the best things about GraphQL is that it works excellently with existing APIs. It's possible to connect any number of existing services to your schema.
+
+The most common source is a REST API. The [`RESTDataSource`](https://www.apollographql.com/docs/apollo-server/features/data-sources.html) is a tool that integrates with `apollo-server` to simplify fetching and caching for existing REST APIs.
+
+Other DataSources are under development, but even without the `DataSource` API, it's possible to connect any backend to a schema. [Resolvers](https://www.apollographql.com/docs/apollo-server/essentials/data.html) can do anything, including fetch data from an SDK or ORM. For more information on how to connect to different sources [read this guide](https://www.apollographql.com/docs/graphql-tools/connectors.html)
```

Comment:
don't link to graphql-tools docs -- they should be going away. do we have docs on extending from the base data source?

Reply:
I don't think we have docs for extending it right now. Definitely worth adding, though.